### DISCUSSION TITLE
quick 1st look

### DISCUSSION THREAD

**Discussion by hari-aum** [SpiralLinux/SpiralLinux-project#13] [General]
greetings...
being a gecko/openSUSE xfce user a while now, today i saw mention of your new spirallinux and why. i tried a live usb, xfce, version.  bootscreen & wallpaper reminiscent of geckolinux.  maybe change with the cool spiral logo & colors, eg, your spirallinux webpage header with shaded reds minus the buttons & text may be one nice choice, :-)
a question: after i did sudo apt update, sudo apt upgrade (choose 'n' - just wanted to see what the upgrades were) i noticed an upgrade to linux-headers-5.10.0-15-common.  a uname -rms in the current live usb spirallinux terminal showed the kernel running as 5.16.0-0.bpo.4-amd64 (a backport kernel i assume).  would you suggest doing the upgrade or holding the bpo kernel or something else...?  thank you.
peace ~ cheers

**Comment by geckolinux** [comment]
Hi there, thanks for checking it out and for the graphical design suggestion, that's obviously not my forte. ;-)

Regarding the updates, nothing bad should happen if you upgrade it. Actually the `linux-headers*` packages are not needed at all, and I will try to remove them from future SpiralLinux builds.
